docker/ubuntu18/Dockerfile

```diff
@@ -32,9 +33,9 @@ RUN apt-get install -y \
   make \
   curl \
   maven \
-  openjdk-8-jdk \
+  openjdk-${jdk}-jdk \
   tzdata
-RUN update-java-alternatives --set java-1.8.0-openjdk-amd64
+RUN update-java-alternatives --set java-1.${jdk}.0-openjdk-amd64
```

Comment:
Ur, is this correct, `java-1.11.0-openjdk-amd64`?

Reply:
I believe so. It properly is set.
```
~/w/I/o/docker ❯❯❯ docker run -it orc-ubuntu18-jdk11 update-java-alternatives -l   
java-1.11.0-openjdk-amd64      1111       /usr/lib/jvm/java-1.11.0-openjdk-amd64

~/w/I/o/docker ❯❯❯ docker run -it orc-ubuntu18-jdk11 java -version  
openjdk version "11.0.10" 2021-01-19
OpenJDK Runtime Environment (build 11.0.10+9-Ubuntu-0ubuntu1.18.04)
OpenJDK 64-Bit Server VM (build 11.0.10+9-Ubuntu-0ubuntu1.18.04, mixed mode, sharing)
```